interfaces/apparmor/template.go

```diff
@@ -342,6 +342,9 @@ var templateCommon = `
   owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/@{SNAP_REVISION}/** wl,
   owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/common/**           wl,
 
+  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/                          r,
+  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/**                        mrkixwl,
```

Comment:
shouldn't this have the same permissions as @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/@{SNAP_REVISION}/ ?

Reply:
Doesn't it? The files under the revisioned dir are affected by:
```
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/**                  mrkix,
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/@{SNAP_REVISION}/** wl,
```
Which looks equivalent to this unless I'm misinterpreting something. However, the directory itself needs to be corrected because the revisioned dir also has `mrkix` permissions and this one doesn't

Reply:
These are not equivalent - note that the existing permissions for `~/.snap/data are`:

```
  # Experimental snap folder changes
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/                    r,
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/**                  mrkix,
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/@{SNAP_REVISION}/** wl,
  owner @{HOME}/.snap/data/@{SNAP_INSTANCE_NAME}/common/**           wl,
```

Whereas you are now using:

```
  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/                          mrkix,
  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/**                        mrkixwl,
```

which allows more permissions in this case (since a snap now can `m`map,loc`k`,and e`x`ecute (with `i`nherit permissions) from the `@{HOME}/Snap/@{SNAP_INSTANCE_NAME}/` directory - which doesn't really make sense for a directory anyway, whereas before it could only `r` from `SNAP_INSTANCE_NAME`).

I think this should look more like:

```
  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/                  r,
  owner @{HOME}/Snap/@{SNAP_INSTANCE_NAME}/**                mrkixwl,
```

ie. read on just the directory but then the ability to create / execute / write arbitrary things within that path.

Reply:
Ah, I see my mistake. Thank you, I've corrected the permissions in the latest commit